packages/react/src/lib/accessibility/Behaviors/Tree/treeBehavior.ts

```diff
@@ -4,16 +4,20 @@ import subtreeBehavior from './subtreeBehavior'
 
 /**
  * @specification
+ * Adds role 'tree' to 'root' component's part.
+ * Adds attribute 'aria-labelledby' based on the property 'aria-labelledby' to 'root' component's part.
  * Embeds component into FocusZone.
  * Provides arrow key navigation in vertical direction.
  * Triggers 'expandSiblings' action with '*' on 'root'.
  */
-const treeBehavior: Accessibility = () => {
+const treeBehavior: Accessibility = props => {
   const subtreeBehaviorData = subtreeBehavior({})
   return {
     attributes: {
       root: {
         ...subtreeBehaviorData.attributes.root,
+        role: 'tree',
+        'aria-labelledby': props['aria-labelledby'],
```

Comment:
since you're using props here, please add `treeBehaviorProps` type

```
type treeBehaviorProps = Pick<AccessibilityAttributes, 'aria-labelledby>
```